src/controllers/roomSocket.ts

```
@@ -52,13 +44,19 @@ function joinRoom(socket: Socket, data: { code: string, joinAs: "player" | "god"
-            await clientDB.collection('rooms').findOne({code: data.code}).then((updatedRoom) => {
+            await clientDB.collection('rooms').findOne({code: data.code}).then(async (updatedRoom) => {
                 socket.join(data.code);
                 socket.emit('rooms:join', updatedRoom);
 
                 socket.to(data.code).emit('rooms:events', updatedRoom);
 
                 console.log(socket.id + ' joined room ' + data.code);
+
+                const room = await clientDB.collection('rooms').findOne({code: data.code});
+
+                if (room && room.creator) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)